lib/screens/play/card/card_actions/card_action_panel.dart

```diff
@@ -119,4 +121,15 @@ class CardActionPanel extends StatelessWidget {
         return Colors.yellow[50];
     }
   }
+
+  IconData _getReplacementIcon(CardActionDetails actionDetails) {
+    switch (actionDetails.replaceType) {
+      case ReplaceType.BACKWARDS:
+        return Icons.arrow_back;
+      case ReplaceType.FORWARDS:
+        return Icons.arrow_forward;
+      default:
+        return Icons.description;
```

Comment:
Ugh, I can't wait for Dart to implement nullable types properly (this would eliminate the need for these pointless `default` branches, because passing `null` would no longer be an option!)